src/views/CooperationInfo.vue

```diff
@@ -371,10 +576,32 @@ export default defineComponent({
       this.$store.dispatch(`${StoreModuleEnum.housesStore}/EDIT_HOUSE`, payload);
       this.closeEditHouseModal();
     },
-    toggle(event: Event) {
+    toggle(event: any, slotProps: any) {
+      console.log(slotProps.quantity_flat);
+      this.house = slotProps;
+      (this.$refs.menu as any).toggle(event);
```

Comment:
you can call toggleConfirm here to prevent code duplicate.

Reply:
removed toggleConfirm since i dont use it anymore